examples/applicationcomposer-jaxws-cdi/pom.xml

```diff
@@ -64,10 +64,10 @@
   </repositories>
   <dependencies>
     <dependency>
-      <groupId>org.apache.tomee</groupId>
-      <artifactId>javaee-api</artifactId>
-      <version>[8.0,)</version>
-      <scope>provided</scope>
+      <groupId>org.apache.tomee.bom</groupId>
+      <artifactId>tomee-plus-api</artifactId>
+      <version>8.0.7-SNAPSHOT</version>
+      <scope>test</scope>
```

Comment:
Shouldn't it be `provided` ?

Reply:
Otherwise:

It seems, that the tests are failing as the `tomee-plus` boms brings in `tomee-security`, which expects some servlet registrations in `TomEESecurityServletAuthenticationMechanismMapper` and subsequently fails with a `NullPointerException`. 

So either

- (a) The test is flawed and requires some additional parameters / registrations
- (b) `TomEESecurityServletAuthenticationMechanismMapper` requires a ` != null` check on `servletRegistrations` in `init(...)`
- (c) Exclude `tomee-security` in the examples dependencies
- (d) Something else (disable security in the example, other things, ...) ....

Maybe @dblevins has some thoughts here?

Reply:
My recommendation would be to for both C and B.  Specifically, do the exclude (B) so we can get the build working, then execute C and remove the exclude.

Reply:
Excluded, thanks, I didn't know whether servletRegistrations should be not null or (more likely) servletRegistrations should not be called

Reply:
@cocorossello The `api` module needs to be in scope `provided` in this example. Besides that: Works fine for me locally :) (and all other examples are passing the tests too... so overall lgtm. thx!)